mediapipe/tasks/ios/core/sources/MPPBaseOptions.m

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import "mediapipe/tasks/ios/core/sources/MPPBaseOptions.h"
+
+@implementation MPPBaseOptions
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.modelAssetFile = [[MPPExternalFile alloc] init];
+  }
+  return self;
+}
+
+- (id)copyWithZone:(NSZone *)zone {
+  MPPBaseOptions *baseOptions = [[MPPBaseOptions alloc] init];
+
+  baseOptions.modelAssetFile = self.modelAssetFile;
```

Comment:
Do we need to do a copy here?

Reply:
All publicly exposed properties throughout the tflite support iOS task library were declared as (copy). This was a design decision taken when we were reviewing the tflite support task library with iOS reviewers at Google. Base Options is a copy property in `MPPTaskOptions` and thus base options needs to confirm to the `NSCopying` protocol.

By the same logic, `MPPExternalFile` should also confirm to it.

Reply:
Sorry, I missed that. No need for an explicit copy here.